ZipBuilder/Package.swift

```diff
@@ -22,7 +22,7 @@ import PackageDescription
 let package = Package(
   name: "ZipBuilder",
   products: [
-    .executable(name: "firebase-pod-updater", targets: ["firebase-pod-updater"]),
+    .executable(name: "FirebaseReleaser", targets: ["FirebaseReleaser"]),
```

Comment:
Targets are recommended to be named in camelcase. The executable name should be command line friendly, so it should be lowercase with dashes e.g.
```
.executable(name: "firebase-releaser", targets: ["FirebaseReleaser"]),
```

Reply:
Done.